agent/utils/ephemeral_ports.go

```diff
@@ -96,13 +98,25 @@ func (pt *safePortTracker) GetLastAssignedHostPort() int {
 
 var tracker safePortTracker
 
+// ResetTracker resets the last assigned host port to 0.
+func ResetTracker() {
```

Comment:
This function `ResetTracker()` has been reviewed in https://github.com/aws/amazon-ecs-agent/pull/3585.